### PR TITLE
Update edx.course.enrollment.upgrade.clicked event in Research Guide

### PR DESCRIPTION
**Pull request by grantgoodmanedX**
## [DOC-3767](https://openedx.atlassian.net/browse/DOC-3767)

I updated the edx.course.enrollment.upgrade.clicked event description in the Research Guide reference. In the History section, I picked Oct 10 as my best guess about when this will get released.
 
### Date Needed 

This was merged today, Oct 6, so presumably will get released early next week.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @robrap 
- [ ] Subject matter expert: 
- [x] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review:

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Squash commits

### CODE REVIEW

File: en_us/data/source/internal_data_formats/tracking_logs/student_event_types.rst [hunk 1]
```diff
@@ -2190,14 +2190,14 @@ different mode is complete.
...
-event when a student clicks this option, and the process of upgrading the
+Students who enroll with a ``student_courseenrollment.mode`` of 'audit' in a
+course that has a verified certificate option see an upgrade link or button
+for the course in various locations. The browser emits this event when a
+student clicks this option, and the process of upgrading the
 ``student_courseenrollment.mode`` for the student to 'verified' begins. See
 :ref:`student_courseenrollment`.
 
-**History**: Added 18 Dec 2013.
+**History**: Added 18 Dec 2013; ``location`` field added 10 Oct 2017.
```
Comment (srpearce) [inline en_us/data/source/internal_data_formats/tracking_logs/student_event_types.rst:2200]: If this field goes live after October 10, make sure to update the date before merging and building.

**Review by srpearce — APPROVED** [review]
👍  with comment re: release date.

**Comment by robrap** [comment]
@grantgoodmanedX: Looks good except for the question I left for @marcotuts in the ticket: DOC-3767.  Thanks.

**Comment by grantgoodmanedX** [comment]
I'm merging this, and will add a further note later about the date-sidebar limitation.

**Pull request merged**